contracts/v3/strategies/MIMConvexStrategy.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import '../interfaces/IConvexVault.sol';
+import '../interfaces/IStableSwap2Pool.sol';
+import './BaseStrategy.sol';
+
+contract MIMConvexStrategy is BaseStrategy {
+    // used for Crv -> weth -> [mim/3crv] -> mimCrv route
+    address public immutable crv;
+    address public immutable cvx;
+
+    address public immutable mim;
+    address public immutable cvx3;
+
+    uint256 public immutable pid;
+    IConvexVault public immutable convexVault;
+    address public immutable mimCvxDepositLP;
+    IConvexRewards public immutable crvRewards;
+    IStableSwap2Pool public immutable stableSwap2Pool;
+
+    constructor(
+        string memory _name,
+        address _want,
```

Comment:
Suggestion: check parameters are not address(0)
See: https://github.com/code-423n4/2021-09-yaxis-findings/issues/35